easybuild/easyconfigs/p/PileOMeth/PileOMeth-0.1.11.eb

```diff
@@ -0,0 +1,40 @@
+##
+#  This is a contribution from Phoenix HPC Service, The University of Adelaide, Australia
+#  Homepage: https://www.adelaide.edu.au/phoenix/
+# 
+#  Copyright::   adelaide.edu.au/phoenix
+#  Authors:: Robert Qiao <robert.qiao@adelaide.edu.au>, Exe Escobedo <exequiel.sepulvedaescobedo@adelaide.edu.au>
+#  License::
+#
+#  Notes::
+###
+
+easyblock = 'MakeCp'
+
+name = 'PileOMeth'
+version = '0.1.11'
+
+homepage = 'https://github.com/dpryan79/PileOMeth'
+description = """PileOMeth processes a coordinate-sorted and indexed BAM or CRAM file containing some form of BS-seq alignments. 
+PileOMeth extracts per-base methylation metrics from them. 
+PileOMeth requires an indexed fasta file containing the reference genome as well.
+"""
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['%(version)s.zip']
+source_urls = ['https://github.com/dpryan79/PileOMeth/archive/']
+
+dependencies = [
+    ('HTSlib', '1.2.1', '', ('foss', '2017a'))
```

Comment:
Since PileOMeth is also being compiled, it's a lot better to specify the `toolchain` to something that is not `dummy`, since then EasyBuild will also set up the build environment (stuff like `$CC`, `$CFLAGS`, etc.)

The dependencies by default will inherit the parent toolchain. You can hard specify to use a particular toolchain on a per-dependency basis, but then these should be a subtoolchain of the parent toolchain. In practice, we only do this when a particular dependency was installed with a `dummy` toolchain (e.g. a binary tool that doesn't require compiling).

See also http://easybuild.readthedocs.io/en/latest/Writing_easyconfig_files.html#dependencies